src/ViewModels/WebMapServiceItemViewModel.js

```diff
@@ -108,8 +108,8 @@ var WebMapServiceItemViewModel = function(application) {
     delete this.__knockoutObservables.dataUrlType;
     knockout.defineProperty(this, 'dataUrlType', {
         get : function() {
-            if (defined(this._dataUrlType)) {
-                return this._dataUrlType;
+            if (defined(this._dataUrl)) {
+                return 'direct';
```

Comment:
No, not all dataUrls should be considered direct links.  Why was this change necessary?